policy-service/src/unit-tests/java/uk/gov/gchq/palisade/service/policy/service/PolicyServiceHierarchyProxyTest.java

```diff
@@ -74,7 +74,9 @@ void testGetRecordLevelRules() {
 
```

Comment:
If you're doing a recursive test, you dont need to check for notNull first

Reply:
Removed the notNull check